### PR TITLE
a user manual entry for the dev team to check out

### PR DESCRIPTION
**Pull request by amyburness**
this is how the user manual needs to be done. for each page and addition you make to the site.

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 28 min

File: docs/src/user/manual/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Screen by screen explanation of
-
+Welcome to the SAWPS manual. This is a page by page guide of the site. Every button, every link and every part of the site is labeled and described in this section. For workflows and how to work with the site pleasse see the user guide.
```
Comment (dimasciput) [inline docs/src/user/manual/index.md:2]:
```suggestion
Welcome to the SAWPS manual. This is a page by page guide of the site. Every button, every link and every part of the site is labeled and described in this section. For workflows and how to work with the site please see the user guide. 
```

File: docs/src/user/manual/profile-page.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# SAWPS profile page
+
+The Profile tab is the first tab on the profile configuration page. The page consists of the following components with the following functionality. 
+
+![Profile Page](img/profile.png)
+
+1. The profile tab - a tab that displays the user profile information
+2. Profile picture - an area that displays the uploaded profile image
+3. Upload profile picture link - Opens the upload profile picture popup (see profile picture pop-up)
+4. Update password button - Opend the update password form for the user to make a new password for the platform (see update password form)
```
Comment (dimasciput) [inline docs/src/user/manual/profile-page.md:10]:
```suggestion
4. Update password button - Opens the update password form for the user to make a new password for the platform (see update password form)
```

**Review by dimasciput — CHANGES_REQUESTED** [review]
Thanks @amyburness , please fix the typo

File: docs/src/user/guide/index.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ To switch to dark mode, simply go to your map and click on the little sun icon w
 
 ![Dark Mode 1](img/dark-mode-1.png)
 
+![test](img/print-3.png)
+
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/index.md:10]: Can this test be removed from the guide, please?

File: docs/src/user/manual/profile-page.md (hunk 2 above)
Comment (amyburness) [reply docs/src/user/manual/profile-page.md:10]: Thanks for the eagle eye. I will update that and PR

File: docs/src/user/guide/index.md (hunk 3 above)
Comment (amyburness) [reply docs/src/user/guide/index.md:10]: Thanks Jeremy completely forgot about that when I was showing Sandhya will remove in PR

File: docs/src/user/manual/img/profile-pick-upload.png
Comment (Jeremy-Prior) [inline docs/src/user/manual/img/profile-pick-upload.png:1]: Please avoid abbreviations in images names, could this be renamed to something like profile-picture-upload or profile-picture-1?

File: docs/src/user/manual/profile-page.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# SAWPS profile page
+
+The Profile tab is the first tab on the profile configuration page. The page consists of the following components with the following functionality. 
+
+![Profile Page](img/profile.png)
+
+1. The profile tab - a tab that displays the user profile information
+2. Profile picture - an area that displays the uploaded profile image
+3. Upload profile picture link - Opens the upload profile picture popup (see profile picture pop-up)
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/profile-page.md:9]: For consistency let's use the non-hyphenated version of popup throughout

```suggestion
3. Upload profile picture link - Opens the upload profile picture popup (see profile picture popup)
```

File: docs/src/user/manual/profile-page.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+7. Title dropdown - a drop-down for the user to select their title which is saved to the database
+8. Email address field - a text field for the user to add their email address which is saved to the database 
+9. Update button - When clicked all changes made to the profile page field are saved to the database
+
+## Profile picture popup
+The profile picture popup appears when the Upload profile picture link is clicked in the profile tab of the profile configuration page. 
+
+![Profile picture popup](img/profile-pick-upload.png)
+
+1. Pop-up title - showing what the popup is for
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/profile-page.md:22]:
```suggestion
1. Popup title - showing what the popup is for
```

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 26 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 26 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 6 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 25 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 27 min

**Comment by amyburness** [comment]
Hi there @dimasciput I updated the changes required what is the blocker on this update

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Review by dimasciput — APPROVED** [review]

**Comment by dimasciput** [comment]
thanks @amyburness

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 22 min

**Pull request merged**